plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/tasks/QueryKBServerTask.java

```diff
@@ -247,25 +245,26 @@ private void putChunkedFileContentInMap(final List<String> fileNamesList) {
         return spotlightPaneTinyEditors;
     }
 
-    private Map<String, ArrayList<CodeInfo>> getProjectNodes(final String esResultJson) {
-        Map<String, String> fileTokensMap = esUtils.getFileTokens(esResultJson);
+    private Map<String, ArrayList<CodeInfo>> getProjectNodes(final String kbResultJson) {
+
+        Map<String, String> fileTokensMap = searchUtils.getFileTokens(kbResultJson);
+
         Map<String, ArrayList<CodeInfo>> pProjectNodes =
                 projectTree.updateProjectNodes(finalImports.keySet(), fileTokensMap);
-        esUtils.updateRepoStarsMap(esResultJson);
+
+        searchUtils.updateRepoStarsMap(kbResultJson);
+
         return pProjectNodes;
     }
 
-    private String getESQueryResultJson() {
-        String esQueryJson = jsonUtils.getESQueryJson(finalImports, windowObjects.getSize(),
-                windowObjects.isIncludeMethods());
-        String esQueryResultJson;
-        if (windowObjects.getLanguage().equals("java")) {
-            esQueryResultJson = esUtils.getESResultJson(esQueryJson,
-                    windowObjects.getEsURL() + JAVA_SEARCH);
-        } else {
-            esQueryResultJson = esUtils.getESResultJson(esQueryJson,
-                    windowObjects.getEsURL() + SCALA_SEARCH);
-        }
-        return esQueryResultJson;
+
+    private String getKodebeagleSearchResultJson() {
+
+        String kbQuery = searchUtils.getQueryJson(finalImports, windowObjects.isIncludeMethods(), 0, windowObjects.getSize() - 1);
+        String kbQueryResultJson = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "kbQueryResultJson". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)